packages/core/src/internal/base/button.base.ts

```diff
@@ -51,6 +58,13 @@ export class CdsBaseButton extends LitElement {
   }
 
   protected updated(props: Map<string, any>) {
+    if (props.has('type') || props.has('name') || props.has('value')) {
+      if (this.marker && this.marker.parentElement) {
```

Comment:
I do not understand what these lines do.

If the `marker` and his parent element exist it will remove the marker and on the next like it sets it again. Why?